composer.json

```diff
@@ -17,19 +17,17 @@
         }
     ],
     "require": {
-        "php": "^5.5.9 || ^7.0",
-        "ajgl/breakpoint-twig-extension": "^0.3.0",
-        "aptoma/twig-markdown": "^2.0",
+        "aptoma/twig-markdown": "^3.0",
         "asm89/twig-cache-extension": "^1.0",
-        "cakephp/cakephp": "^3.2",
-        "jasny/twig-extensions": "^1.0",
-        "twig/twig": "^1.18"
+        "cakephp/cakephp": "^3.5.1",
+        "twig/twig": "^2.4.3",
+        "php": "^7.0"
```

Comment:
Could you put the PHP requirement at the top of the list?

Reply:
Sorted it alphabetically. Will fix

Reply:
Cheers, when you let composer do that `php` always ends on top